conbench/entities/history.py

```diff
@@ -481,6 +488,9 @@ def _add_rolling_stats_columns_to_df(
         for x in df["change_annotations"]
     ]
 
+    # Add in step changes automatically detected
+    df["begins_distribution_change"] = df["begins_distribution_change"] | df["is_step"]
```

Comment:
Nice, wow, if this works well we won't need to manually curate the data anymore, and therefore we could probably take away the DISTRIBUTION_COMMITS max window size I talked about elsewhere on this PR. That could be cool.

Reply:
Yeah, this is a little change with a big impact. It looks good so far, but some more testing before we let this live may be a good idea.

Reply:
Two people are wowed, I don't know what this is doing w/o reading further docs. Can we add a come comment so I can have a quicker wow effect, too? :)

Reply:
>  if this works well we won't need to manually curate the data anymore

Ah. I think I slowly get this; this is about automatic distribution change detection. I didn't expect this as part of this PR, that's also probably what Austin didn't expect, and therefore the surprise moment.